ts/components/wallet/TransactionsList.tsx

```diff
@@ -86,7 +87,13 @@ export class TransactionsList extends React.Component<Props, State> {
             removeClippedSubviews={false}
             dataArray={ops as any[]} // tslint:disable-line
             renderRow={(item): React.ReactElement<any> => (
-              <ListItem onPress={(): boolean => navigate("")}>
+              <ListItem
+                onPress={(): boolean =>
```

Comment:
@fpersico is it good to trigger navigation from the component or we'd better use a generic action that gets handled in a saga ?

Reply:
In this case is good. If you have to check something or perform any side effect before navigation using saga is better.